### PR TITLE
Update contributing doc and mention requirements to build the project

### PR DESCRIPTION
**Pull request by dbuzhorkaa**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,28 +1,64 @@
-# How to Contribute
+====================
+Contributing to CDAP.io website
+====================
```
Comment (bdmogal) [inline CONTRIBUTING.md]: this (and line 1) should extend to the end of the word "website"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,28 +1,64 @@
...
+  sure to post a comment after pushing. The new commits will show up in the pull
+  request automatically, but the reviewers will not be notified unless you comment. 
+
+* Committers use LGTM ("Looks Good To Me") in comments on the code review to indicate acceptance. 
+  The code will be merged after the reviewer comments a "LGTM" on the pull request.
+
+License and Trademarks
+======================
+
+Copyright © 2015 Cask Data, Inc.
```
Comment (bdmogal) [inline CONTRIBUTING.md]: update year

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,28 +1,64 @@
...
+Add a link to your PR on the issue, and in an email to `cdap-dev@googlegroups.com <mailto:cdap-dev@googlegroups.com>`__.
 
-## Community Guidelines
+Conventions
+-----------
+* Fork the repository and make changes on your fork in a feature branch.
 
-This project follows
-[Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+* Write clean code. Uniformity in formatting code promotes ease of writing, reading, and maintenance.
```
Comment (bdmogal) [inline CONTRIBUTING.md]: make sure that this does not render bold.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,28 +1,64 @@
...
+If your pull request is not accepted on the first try, don't be discouraged. 
+If there is a problem with the implementation, you will receive feedback on what to improve.
+Add a link to your PR on the issue, and in an email to `cdap-dev@googlegroups.com <mailto:cdap-dev@googlegroups.com>`__.
 
-## Community Guidelines
+Conventions
+-----------
+* Fork the repository and make changes on your fork in a feature branch.
 
-This project follows
```
Comment (bdmogal) [inline CONTRIBUTING.md]: I think it might be ok to leave the community guidelines section as a separate section.

**Comment by dbuzhorkaa** [comment]
@bdmogal I addressed your comments. Please take another look.

**Comment by bdmogal** [comment]
@dbuzhorkaa can you squash the commits? then I will merge.

**Pull request merged**